instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/network/NetworkAttributesExtractor.java

```diff
@@ -32,13 +33,12 @@ public static <REQUEST, RESPONSE> NetworkAttributesExtractor<REQUEST, RESPONSE>
   private final InternalNetworkAttributesExtractor<REQUEST, RESPONSE> internalExtractor;
 
   NetworkAttributesExtractor(NetworkAttributesGetter<REQUEST, RESPONSE> getter) {
-    // the NetworkAttributesExtractor will always emit new semconv
     internalExtractor =
         new InternalNetworkAttributesExtractor<>(
             getter,
             NetworkTransportFilter.alwaysTrue(),
-            /* emitStableUrlAttributes= */ true,
-            /* emitOldHttpAttributes= */ false);
+            SemconvStability.emitStableHttpSemconv(),
+            SemconvStability.emitOldHttpSemconv());
```

Comment:
I've decided to have the new extractors emit the old spec as well: this will ease the transition from the old `Net*` classes, and will reduce the `@SuppressWarnings("deprecation")` annotations in our codebase.

The only attributes that are not covered by the new extractors are `net.transport` and `net.sock.family` (because the new attributes have different values), but there's like ~5 instrumentations using these anyway, so we'll just suppress deprecations there.